other-docs/guides/upgrading/v2.md

```diff
@@ -0,0 +1,103 @@
+# Upgrading to v2
+
+## Breaking Changes
+
+There are no known breaking changes in version 2.
+
+
```

Comment:
Is it worth adding an "update `^1.0.0`, to `2.0.0` and deploy to upgrade to Altis version 2."

Reply:
Yeah, although that's already covered under the main upgrading guide so didn't think it was worth repeating here.